src/iamService.ts

```diff
@@ -29,6 +29,7 @@ import * as fallback from './fallback';
 import * as path from 'path';
```

Comment:
I think this is maybe fallout from removing the old loader code? (the unused warning)

Reply:
Thank you! Fixed.